run3/flow/ML/run_cutvar_test.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+export config_flow="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/config_flow_d0_test_ml.yml"
+export anres_dir="/media/wuct/wulby/ALICE/AnRes/D0_flow/pass4/ML/Results/AnalysisResults_BDT_DATA_test.root"
+export output_dir="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/ML/test"
+export cent="k3050"
+export vn_method="sp"
+export res_file="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/Results/2060/k3050/small/sp/resolution/resosp3050s_291131_inte_gain_Reso.root"
+export suffix="test"
+export isMC="--isMC"
+
+export MakeyamlPath="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/ML/make_yaml_for_ml.py"
+export SimFitPath="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/get_vn_vs_mass.py"
+export ProjPath="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/ML/proj_thn_mc.py"
+export EffPath="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/compute_efficiency.py"
+export CurVarFracPath="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/ML/compute_frac_cut_var.py"
+export DataDrivenFracPath="/home/wuct/ALICE/local/DmesonAnalysis/ComputeDataDrivenFraction.py"
+export v2vsFDFracPath="/home/wuct/ALICE/local/DmesonAnalysis/run3/flow/ML/ComputeV2vsFDFrac.py"
+
+export output_dir="${output_dir}/cutvar_${suffix}"
+
+# make yaml file
+echo -e "\e[34mpython3 make_yaml.py ${config_flow} -o ${output_dir} -s ${suffix}\e[0m"
+python3 ${MakeyamlPath} ${config_flow} -o ${output_dir} -s ${suffix}
+
+# Get the number of projection files for different cutsets
+CutVarNum=$(find "${output_dir}/config" -type f -name "cutset*.yml" | wc -l)
+echo "Number of cutsets: ${CutVarNum}"
+
+# # Cut variation (aply the cut and project)
```

Comment:
With this script, you first produce all the yaml configurations you want to test and then run a cut variation on them. It seems to me a bit redundant, I would keep the cut variation script for quick tests, and I would use this to perform the cut variation down to the final results